cleanlab/count.py

```diff
@@ -90,6 +93,8 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
       Entry ``(j, k)`` in the matrix is the number of examples confidently counted into the pair of ``(noisy label=j, true label=k)`` classes.
       The `confident_joint` can be computed using :py:func:`count.compute_confident_joint <cleanlab.count.compute_confident_joint>`.
       If not provided, it is computed from the given (noisy) `labels` and `pred_probs`.
+      If `multi_label` is True, then the `confident_joint` should be a one-vs-rest array of shape ``(K, 2, 2)``, refer to filter.find_label_issues() for detailsf.
```

Comment:
typo and need to link to function properly

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/a684583aa0e89de6414172c891b1db305e31ade3/cleanlab/count.py#L96